### PR TITLE
{tools}[giolf/2017b] HPL v2.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
New toolchain using GCC, IntelMPI, OpenBLAS, LAPACK, ScaLAPACK and FFTW.

### CODE REVIEW

**Comment by akesandgren** [comment]
This of course requires https://github.com/easybuilders/easybuild-framework/pull/2310

File: easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-gimpi-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+toolchain = {'name': 'gimpi', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = ['fftw-%(version)s-pl2.tar.gz']
+checksums = ['a5de35c5c824a78a058ca54278c706cdf3d4abba1c56b63531c2cb05f5d57da2']
+
+skipsteps = ['test']
+#runtest = 'check'
```
Comment (boegel) [inline easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-gimpi-2017b.eb]: @akesandgren Why is this here?

Comment (boegel) [inline easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-gimpi-2017b.eb]: related to https://github.com/easybuilders/easybuild-easyblocks/issues/1100?

Comment (akesandgren) [reply easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-gimpi-2017b.eb]: oops, forgot to remove it. Had problems with the mpi tests on my laptop for some reason.

Comment (akesandgren) [reply easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-gimpi-2017b.eb]: Yes.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 6 (6 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/23ab92de15a15494c1fb020fd36ddff7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c62409822f6250c23a53a287252a14ae for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2412.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/52fb53f794d1ef97e69288a65b830209 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!
